test/test_webservice.py

```diff
@@ -61,6 +61,7 @@
     "proxy_username": 'user',
     "proxy_password": 'password',
     "network_transfer_timeout_seconds": 30,
+    "network_cache_size_bytes": 100*1000*1000,
```

Comment:
This would need to be changed too plus similar lines below.

Reply:
I think we already had this discussion (https://github.com/metabrainz/picard/pull/122/files#r4631209).

MB stands for Megabytes and SI prefixes defines Mega as 10⁶. MiB could be used of course, but I guess end user doesn't much about the difference between Mebibytes and Megabytes anyway.

https://en.wikipedia.org/wiki/Megabyte
https://physics.nist.gov/cuu/Units/binary.html
https://www.bipm.org/en/measurement-units/si-prefixes

My preference is to stick to widely adopted International System of Units and their prefixes.